libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -22,24 +22,23 @@ type ExplorerParams = {
 
 class BitcoinLikeExplorer implements IExplorer {
   client: Pool<{ client: AxiosInstance }>;
-
   underlyingClient: AxiosInstance;
-
   disableBatchSize = false;
 
-  explorerVersion: "v2" | "v3";
-
   constructor({
-    explorerURI,
-    explorerVersion,
+    cryptoCurrency,
     disableBatchSize,
+    forcedExplorerURI,
```

Comment:
why is it called "forced" ? Could just be named `explorerURI`, it's an optional value

Reply:
Because it enforces an explorerURI instead of using the default one provided by the cryptoCurrency